src/commands/test/lib/run-tests-command.ts

```diff
@@ -199,4 +205,16 @@ export class RunTestsCommand extends AppCommand {
     let parsedParameters = parseTestParameters(this.testParameters);
     _.merge(manifest.testFramework.data, parsedParameters || {});
   }
+
+  private async processLanguage(): Promise<void> {
+    if (!this.language) {
+      return;
+    }
+    for (let index in this.testParameters) {
+      if (this.testParameters[index].indexOf("language=") === 0) {
+        throw new Error("Argument '--language' cannot be used with a 'language' test parameter");
+      }
+    }
+    this.testParameters.push("language=" + this.language);
```

Comment:
How is this shipped as a json payload? AFAIK using a testParameter is not working, but I'm not 100% sure. `--language` should be transferred in the same way as `--locale` i.e. not embedded in the `testParameters` but at top level